src/paho/mqtt/client.py

```diff
@@ -1345,6 +1345,14 @@ def enable_bridge_mode(self):
         """
         self._client_mode = MQTT_BRIDGE
 
+    def isConnected(self):
```

Comment:
Could you just rename this method "is_connected" ? Python style (PEP-8) recommends using snake case for method name.

Reply:
I have done the changes as requested. Will follow Python styling going forward. Thanks.